src/Materials/Textures/dynamicTexture.ts

```diff
@@ -45,7 +45,7 @@ export class DynamicTexture extends Texture {
             this._canvas = options;

```

Comment:
we end up down this path for AdvancedDynamicTexture.CreateFullscreenUI. it generates exceptions because we submit a 0x0 texture size which bgfx doesn't support. we may want to check that the options are a canvas element compared to assuming they are a canvas element because they have a getContext function

Reply:
0x0 texture should not crash when calling `AdvancedDynamicTexture.CreateFullscreenUI`. I guess the layer will resize the texture later at runtime. NativeCanvas should discard texture creation and rendering until a valid size is provided.